examples/MultiTemplates_SAS_SodiumLoop/sas_template

```diff
@@ -0,0 +1,199 @@
+

```

Comment:
the name of the example folder should be changed for consistency purposes - `1App_SAS_MultiTemplate-SodiumLoop`